docs/tutorials/data_pipeline.md

```diff
@@ -146,28 +146,29 @@ For each operation, we list the related dict fields that are added/updated/remov
 
 ## Extend and use custom pipelines
 
-1. Write a new pipeline in any file, e.g., `my_pipeline.py`. It takes a dict as input and return a dict.
+1. Write a new pipeline e.g., `my_pipeline.py`. It takes a dict as input and returns a dict.
```

Comment:
suggest to rephrase it as "Write a new pipeline in a file, e.g., in `my_pipeline.py`."